docs/self-managed/concepts/document-handling/configuration.md

```diff
@@ -0,0 +1,167 @@
+---
+id: document-handling-configuration
+title: "Configuration and storage"
+description: "Learn more about storage configuration options like Google Cloud Platform, AWS S3, local folders, and in-memory."
+keywords: ["document handling"]
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+![document handling storage options](../img/document-handling-config.png)
+
+As shown in the screenshot above, there are several storage options:
+
+- A [**Google Cloud Platform**](https://cloud.google.com/storage) bucket storage integration can be used in Self-Managed configurations with Camunda 8 Run and Helm.
+- Use [**AWS S3**](https://aws.amazon.com/s3/) storage and bucket creation per cluster to securely store and retrieve documents in an external, scalable storage solution for Self-Managed, and to ensure storage is properly isolated and managed for each environment. This can be used with Camunda 8 Run and Helm.
+- Documents can be stored in **local folders**. This can be used only for Camunda 8 run, and is not supported for production environments.
+- Documents can be stored **in-memory**. If the application is stopped, the document will be lost. This can be used with both Camunda 8 Run and Helm, but is not supported for production environments.
+
+:::note
+GCP and AWS are supported for Self-Managed in production. Self-Managed users may configure in-memory and local storage using [Camunda 8 Run](/self-managed/setup/deploy/local/c8run.md).
+:::
+
+:::note
+If no configuration is provided, the default document storage is **in-memory**. To change this to a different storage method, use the environment variables in the section below for **every** component using document handling. No additional configuration is required for in-memory storage.
+:::
+
+To set what storage should be used, accepted values for `DOCUMENT_DEFAULT_STORE_ID` are `aws`, `in-memory`, `gcp` (for Google Cloud Platform), and `local` (for local storage).
```

Comment:
@christinaausley did I understand you correctly that if we are talking about config values, `inmemory` _with no dash_ is correct?

Reply:
@christinaausley this still needs to be fixed. Either this dash needs to be removed, or the sentence needs to be reworked. Right now it reads like this is a list of the possible accepted values, which means `inmemory` is the right format for this list.